incubator/monochart/values.yaml

```
@@ -314,3 +314,28 @@ ingress:
 #    - secretName: server-tls
 #      hosts:
 #      - domain.com
+
+# Istio gateway
+istio_gateways:
```

Should we add `[]` since the list is commented out?
```suggestion
istio_gateways: []
```

---

@goruha 👆